plugin.rb

```diff
@@ -14,28 +14,15 @@ module ::DiscourseCustomTopicLists
 end
 
 require_relative "lib/discourse_custom_topic_lists/engine"
+require_relative "lib/discourse_custom_topic_lists/custom_topic_list"
 register_asset "stylesheets/common/common.scss"
 
 after_initialize do
+  require_relative "app/controllers/discourse_custom_topic_lists/lists_controller"
+
+  Discourse::Application.routes.append { mount ::DiscourseCustomTopicLists::Engine, at: "" }
```

Comment:
I'm curious why this change is necessary. The previous line to mount the engine's route in `config/routes.rb` should work.